pandas/core/tools/timedeltas.py

```diff
@@ -18,26 +18,36 @@
 
 def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     """
-    Convert argument to timedelta
+    Convert argument to timedelta.
+
+    Timedeltas are absolute differences in times, expressed in difference
+    units e.g. days, hours, minutes, seconds. This method converts an argument
+    from a recognized timedelta format / value into a Timedelta type.
 
     Parameters
     ----------
-    arg : string, timedelta, list, tuple, 1-d array, or Series
-    unit : unit of the arg (D,h,m,s,ms,us,ns) denote the unit, which is an
-        integer/float number
-    box : boolean, default True
-        - If True returns a Timedelta/TimedeltaIndex of the results
+    arg : str, timedelta, list, tuple, 1-d array, or Series
+          The argument which needs to be converted to timedelta.
+    unit : str, default 'ns'
+          Denotes the unit (D,h,m,s,ms,us,ns) of the arg.
+    box : bool, default True
+        - If True returns a Timedelta/TimedeltaIndex of the results.
         - if False returns a np.timedelta64 or ndarray of values of dtype
-          timedelta64[ns]
+          timedelta64[ns].
     errors : {'ignore', 'raise', 'coerce'}, default 'raise'
-        - If 'raise', then invalid parsing will raise an exception
-        - If 'coerce', then invalid parsing will be set as NaT
-        - If 'ignore', then invalid parsing will return the input
+        - If 'raise', then invalid parsing will raise an exception.
+        - If 'coerce', then invalid parsing will be set as NaT.
+        - If 'ignore', then invalid parsing will return the input.
 
     Returns
     -------
     ret : timedelta64/arrays of timedelta64 if parsing succeeded
```

Comment:
you can get rid of the `ret`, and leave here just the types `timedelta64 or numpy.array of timedelta64`. We should be able to automatically parse the line with the types, so no description should go there. A description on what is being returned, and clarification on when every type is being returned should go in the next line (indented).